external/llvm-project/mlir/include/mlir/Dialect/MemRef/IR/MemRefOps.td

```diff
@@ -1221,9 +1224,29 @@ def LoadOp : MemRef_Op<"load",
   let arguments = (ins Arg<AnyMemRef, "the reference to load from",
                            [MemRead]>:$memref,
                        Variadic<Index>:$indices,
-                       DefaultValuedOptionalAttr<BoolAttr, "false">:$nontemporal);
+                       DefaultValuedOptionalAttr<BoolAttr, "false">:$nontemporal,
+                       OptionalAttr<LLVM_AliasScopeArrayAttr>:$alias_scopes,
```

Comment:
... that feels like it'll cause screaming.

We might want to RFC up an MLIR version of these annotations

Reply:
My other theory here is to let these be discardable attributes - use `llvm.alias_scopes = ...`

That should be more reasonable and we can argue that, hey, these are advisory annotations and if a pass doesn't preserve them, well, that's life until someone designs a better scheme for this